src/index.html

```diff
@@ -1,13 +1,281 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header page__section">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+            <a class="logo">
+              <img src="./images/logo.svg"
+                   alt="bose"
+                   class="logo__img"
+              >
```

Comment:
fix intends in all places
```suggestion
              <img
                src="./images/logo.svg"
                alt="bose"
                class="logo__img"
              >
```